test/Project.toml

```diff
@@ -0,0 +1,2 @@
+[deps]
```

Comment:
I would prefer us to keep the `target` stuff in the Project because this way of doing it is explicitly marked experimental.